coralproject/talk
# Embed link

## What does this PR do?

Adds the appropriate embed link to the settings page in `coral-admin`.

## How do I test this PR?

1. `npm run build`
2. `npm start`
3. Go to `localhost:3000/admin
4. Configure > Embed Comment stream
5. The embed text should be present.
6. The Copy button should copy to your clipboard.

## Note

I added some tweaks to the `coral-admin` build process to better tie it to the centrally run `npm build` command.

@coralproject/tech
